src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -237,6 +237,32 @@ protected final void verifyWithInlineConfigParser(String filePath, String... exp
         verify(parsedConfig, filePath, expected);
     }
 
+    /**
+     * Performs verification of two files with the given file paths using specified
+     * configuration and the array expected messages. Also performs verification of the config
+     * specified in the input file.
```

Comment:
Please add a bit more detail here (especially about using only the first config), and explain why someone would need to use this method vs the existing verifyWithInline... methods.